labconnect/__init__.py

```diff
@@ -4,6 +4,7 @@
 
 # Import Flask modules
 from flask import Flask
+from flask_cors import CORS
```

Comment:
can you remove all these additions are aren't using this right now. Any instance of CORS should be removed in this file